src/.eslintrc.yml

```diff
@@ -8,6 +8,10 @@ extends:
 parser: '@typescript-eslint/parser'
 rules:
   no-inner-declarations: off
+  indent:
+    - error
+    - 4
+    - SwitchCase: 1
```

Comment:
Without this, `eslint` will enfore:
```js
switch (condition) {
case 'hello':
  return 'world';
default:
  return 'unknown';
}
```
But I feel this is a bit nicer:
```js
switch (condition) {
    case 'hello':
        return 'world';
    default:
        return 'unknown';
}
```

Any oppositions?

Reply:
I prefer the latter as well